### ISSUE TITLE
Add-on problem

### ISSUE THREAD

**Issue by isabellaalstrom** [cgarwood/homeassistant-zwave_mqtt#55]
Didn't know if you wanted add-on issues here as well, point me in another direction if not.
My add-on seems to start at first but this is the log:

```[INFO] Setup Hass.io mqtt service to core-mosquitto.local.hass.io
2020-02-05 14:36:20,617 CRIT Supervisor is running as root.  Privileges were not dropped because no user is specified in the config file.  If you intend to run as root, you can set user=root in the config file to avoid this message.
2020-02-05 14:36:20,637 INFO supervisord started with pid 6
2020-02-05 14:36:21,646 INFO spawned: 'websockify' with pid 31
2020-02-05 14:36:21,655 INFO spawned: 'ozwadmin' with pid 32
2020-02-05 14:36:21,661 INFO spawned: 'xvfb' with pid 33
2020-02-05 14:36:21,666 INFO spawned: 'mosquitto' with pid 34
2020-02-05 14:36:21,673 INFO spawned: 'x11vnc' with pid 35
2020-02-05 14:36:21,679 INFO spawned: 'ozwdaemon' with pid 36
2020-02-05 14:36:21,697 INFO exited: ozwadmin (exit status 127; not expected)
2020-02-05 14:36:22,889 INFO success: websockify entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2020-02-05 14:36:22,895 INFO spawned: 'ozwadmin' with pid 39
2020-02-05 14:36:22,896 INFO success: xvfb entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2020-02-05 14:36:22,896 INFO success: mosquitto entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2020-02-05 14:36:22,896 INFO success: x11vnc entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2020-02-05 14:36:22,897 INFO success: ozwdaemon entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2020-02-05 14:36:22,919 INFO exited: ozwadmin (exit status 127; not expected)
2020-02-05 14:36:25,871 INFO spawned: 'ozwadmin' with pid 46
2020-02-05 14:36:25,895 INFO exited: ozwadmin (exit status 127; not expected)
2020-02-05 14:36:28,942 INFO spawned: 'ozwadmin' with pid 52
2020-02-05 14:36:28,969 INFO exited: ozwadmin (exit status 127; not expected)
2020-02-05 14:36:29,971 INFO gave up: ozwadmin entered FATAL state, too many start retries too quickly
```

I have tried rebuilding, restarting add-on and restarting hass. The add-on keeps on running and I can open web ui but it's only black.

**Comment by daanvb** [comment]
Exact same thing happening to me

**Comment by marcelveldt** [comment]
Have you spotted the add-on settings and in particular the path to the Z-wave device.
It's not (yet) auto detected so you will have to fill in the path to the serial device.

BTW: It looks like OZWAdmin is causing the error. I will remove that from the image for the time being as that component is really in early stage. It's a management console to directly manage advanced properties of your Z-Wave mesh and controllers.

We can set-up an additional add-on just for testing the OZWAdmin once it's more usable.

**Comment by marcelveldt** [comment]
@isabellaalstrom and @daanvb the new version is up on the repo. Can you give it another try ?

**Comment by isabellaalstrom** [comment]
I will try it tomorrow as soon as I get to work (it's on our office hass) and get back to you.

**Comment by daanvb** [comment]
> Have you spotted the add-on settings and in particular the path to the Z-wave device.
It's not (yet) auto detected so you will have to fill in the path to the serial device.
> 
> @isabellaalstrom and @daanvb the new version is up on the repo. Can you give it another try ?

Yeah I spotted that. My Zwave is at \dev\ttyACM1

I too will give it another go tomorrow. Bit zwave'd out for today! Do i just reinstall v0.0.3 via HACS?

**Comment by marcelveldt** [comment]
> > Have you spotted the add-on settings and in particular the path to the Z-wave device.
> > It's not (yet) auto detected so you will have to fill in the path to the serial device.
> > @isabellaalstrom and @daanvb the new version is up on the repo. Can you give it another try ?
> 
> Yeah I spotted that. My Zwave is at \dev\ttyACM1
> 
> I too will give it another go tomorrow. Bit zwave'd out for today! Do i just reinstall v0.0.3 via HACS?

I've updated the add-on (hassio/supervisor)

**Comment by balloob** [comment]
Reminder: Go to Supervisor UI -> System and click on Hardware info to see the path of your Z-Wave stick. (is this in the docs yet?)

**Comment by isabellaalstrom** [comment]
Add-on now starting up fine! I'm not really sure how to proceed with testing though, is everything done by calling services?

**Comment by daanvb** [comment]
Working here too, thanks. All devices and most entities discovered. Missing the motion sensor on one of my sensors for some reason.

Guessing there is no control panel yet?

**Comment by marcelveldt** [comment]
Control panel is not yet there. You can do the commands manually by using the services or even directly on the MQTT bus. What commands/configuration do you need most ?

**Comment by marcelveldt** [comment]
> Working here too, thanks. All devices and most entities discovered. Missing the motion sensor on one of my sensors for some reason.
> 

The initial scan takes a very long time because all battery nodes need to be awake once to discover their info. This is just only the first time only. In my own situation I got tired of waiting and woke up each pir sensor myself by pressing the button.

**Comment by daanvb** [comment]
I don't think it is anything to do with this addon why it is missing. All my devices are connected to mains power. My zwave has always been a bit flakey. I have been using zwave2mqtt addon and then trying to use ozwcp to configure but i have been finding that some settings such as offsets, reporting and motion detection time weren't being stored....I think the whole system needs a full factory reset and to start again....

Ah yeah I have found the services, just seen the set_parameter_config....

if i wanted to try and set a temperature offset on my Aoetec MultiSensor 6 
would i call the service like this

```
node_id: 21
parameter: 201 #201 being the temperature calibration parameter from specs
value: -15
```

**Comment by isabellaalstrom** [comment]
Managed to add a node now and something showed up in home assistant, but the add-on keeps giving me this:

```2020-02-06 11:21:57,940 INFO success: ozwdaemon entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2020-02-06 11:22:18,368 INFO exited: ozwdaemon (exit status 255; not expected)
2020-02-06 11:22:19,378 INFO spawned: 'ozwdaemon' with pid 104
2020-02-06 11:22:20,895 INFO success: ozwdaemon entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2020-02-06 11:22:41,316 INFO exited: ozwdaemon (exit status 255; not expected)
2020-02-06 11:22:41,322 INFO spawned: 'ozwdaemon' with pid 109
2020-02-06 11:22:42,820 INFO success: ozwdaemon entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2020-02-06 11:23:03,254 INFO exited: ozwdaemon (exit status 255; not expected)
2020-02-06 11:23:03,260 INFO spawned: 'ozwdaemon' with pid 114
2020-02-06 11:23:04,757 INFO success: ozwdaemon entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
```


![image](https://user-images.githubusercontent.com/22006797/73928546-926ae600-48d3-11ea-8ff9-9be4aa1e281f.png)

**Comment by isabellaalstrom** [comment]
Maybe I should try to reset everything. I am in a good place for testing since there's nothing running in production on this yet. I'm only just starting out with Home Assistant at my workplace :)

**Comment by Fishwaldo** [comment]
I just pushed a fix for OpenZWave that was causing crashes. Docker Images are rebuilding and should be available in a few hours. Try again then.

**Comment by marcelveldt** [comment]
> I just pushed a fix for OpenZWave that was causing crashes. Docker Images are rebuilding and should be available in a few hours. Try again then.

Great news! Will this also fix the crash with the set value on a newly added dimmer ?

**Comment by Fishwaldo** [comment]
Yes. Hopefully

**Comment by daanvb** [comment]
> 
> 
> I just pushed a fix for OpenZWave that was causing crashes. Docker Images are rebuilding and should be available in a few hours. Try again then.

Will the Hassio daemon add-on be updated with this new image?

**Comment by marcelveldt** [comment]
> > I just pushed a fix for OpenZWave that was causing crashes. Docker Images are rebuilding and should be available in a few hours. Try again then.
> 
> Will the Hassio daemon add-on be updated with this new image?

Yes, a new version of the daemon add-on is now published (1.0.10)

**Comment by balloob** [comment]
Alright, closing this issue. Please open a new one if you see more problems 👍
